src/lambdas/dispatcher/src/lambda_function.py

```diff
@@ -64,7 +63,7 @@ def _analysis(command: AnalysisCommand, client: DataApiClient) -> List[Dict[str,
 
     for table in command.parameters.tables:
         asset_uri = client.get_1x1_asset(table.dataset, table.version)
-        if table.analysis in [Analysis.viirs, Analysis.modis]:
+        if table.analysis in [Analysis.viirs, Analysis.modis, Analysis.burned_areas]:
```

Comment:
I have seen this Analysis combination at least 3 times. Is is worth, making it a Global?

Reply:
done